docs/04-reference/wingcli.md

```diff
@@ -0,0 +1,154 @@
+---
+title: CLI
+id: cli
+description: Wing CLI Reference
+keywords: [Wing reference, Wing language, language, Wing language spec, Wing programming language, cli]
+---
+
+The Wing CLI is a command line interface for Wing. It is used to compile, test, and run Wing
+programs, and we hope it will become your best friend.
+
+## Installation
+
+The CLI is distributed as an npm package which can be installed globally using:
+
+```bash
+npm i -g winglang
+```
+
+The usage is:
+
+```bash
+$ wing <command> <options>
+```
+
+## Run: `wing run` / `wing it`
```

Comment:
We can also mention the default entrypoint behavior

Reply:
Done